apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -1086,8 +1086,7 @@ get_info(Peer, Type) ->
 	case get_info(Peer) of
 		info_unavailable -> info_unavailable;
 		Info ->
-			{Type, X} = lists:keyfind(Type, 1, Info),
-			X
+			maps:get(atom_to_binary(Type), Info)
```

Comment:
The previous implementation handled the case when an expected key would not be sent (plus the absence of the release field treated as release=0), here we will have badarg, if I see it right.

Reply:
what about if I change to `maps:get(atom_to_binary(Type), Info, info_unavailable)`?

I believe that's what happened previously if the expected key was not sent.

In that case the only situation that isn't handled is the special handling for a missing `release`. Which perhaps I can handle in `ar_peers.erl` (the only place where `get_info(Peer, release)` is called?

My motivation in changing this to a map was to cut down on some of the boilerplate that we had where we converted JSON -> JSonStruct -> Map -> value.

Let me build a commit to show you the proposed fix.

Reply:
Proposed fix: https://github.com/ArweaveTeam/arweave/pull/581/commits/319984a85aba444a1492271f22f53ba8010f8898

Reply:
Makes sense, looks good, thank you!